users/models/follower.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+class Follower(models.Model):
+    follower_id = models.ForeignKey(User, on_delete=models.CASCADE, related_name='follower_of')
+    followed_id = models.ForeignKey(User, on_delete=models.CASCADE, related_name='followed_by')
```

Comment:
Llamaría al campo solo followed, por simplicidad.

Reply:
clave foranea

Reply:
Sí, pero en este caso no se cumple la norma que indicabas de que el campo se llama como el modelo al que se enlaza con clave foránea. El modelo es User, no Followed. Por eso digo que, se siga el convenio que se siga, que sea consistente. Quitar los sufijos _id me parece lo más simple para no tener que recordar cuando se usan y cuando no con las claves foráneas.